app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
```

Comment:
Fix formatting: brackets must be located in one of two options: open and start with the text or have a line break between the text

Reply:
Yep